trello_api.py

```diff
@@ -2,20 +2,73 @@
 import os
 from copy import deepcopy
 from to_do_item import ToDoItem
+from to_do_item import Status
+from dotenv import find_dotenv, load_dotenv
 
 API_KEY = os.getenv('API_KEY')
 TOKEN = os.getenv('TOKEN')
 BOARD_ID = os.getenv('BOARD_ID')
-TO_DO_LIST_ID = os.getenv('TO_DO_LIST_ID')
-DONE_LIST_ID = os.getenv('DONE_LIST_ID')
 
 CREDENTIALS = { 'key': API_KEY, 'token': TOKEN }
 
 TRELLO_API_BASE_URL = 'https://api.trello.com/1'
 
+TO_DO_LIST_ID = None
+DOING_LIST_ID = None
+DONE_LIST_ID = None
+
+def reload_credentials():
+    global CREDENTIALS
+    CREDENTIALS = { 'key': API_KEY, 'token': TOKEN }
+
+def load_api_key_if_none():
+    global API_KEY
```

Comment:
You should avoid the use of globals as its lifetime is not terribly long for a WSGI application (likely the lifetime of a single request).